src/components/AnalyticsDrawer/AnalyticsDrawer.tsx

```diff
@@ -1,30 +1,52 @@
+import CircularProgress from 'material-ui/CircularProgress';
 import { Drawer } from 'material-ui'
 import MarkerIcon from 'mui-icons/cmdi/map-marker'
 import * as React from 'react'
 import { Link } from 'react-router-dom'
-import { withStore } from '../../services/store'
+import { store, withStore } from '../../services/store'
 import { InfoBox } from '../InfoBox/InfoBox'
 import { ServiceAreaSelector } from '../ServiceAreaSelector/ServiceAreaSelector'
+import { Refresh } from '../Refresh/Refresh'
 import './AnalyticsDrawer.css'
 import { ServiceAreaAnalytics } from './ServiceAreaAnalytics'
 import { TotalAnalytics } from './TotalAnalytics'
 
+function pickAnalytics() {
```

Comment:
Consider making this a React component, or inlining it into `AnalyticsDrawer`. It's an antipattern to import `store` and depend on it directly (we should probably not export it in the first place).

Injecting `store` is a kind of [dependency injection](https://en.wikipedia.org/wiki/Dependency_injection), and is a less hacky version of what Redux uses `context` for. Because `store` is stateful and we don't want to `import` stateful things directly (because 1. it makes them hard to test, and 2. it makes it hard to make them not-singletons in the future), it's much better to use `withStore` instead.